### PR TITLE
Sample migration 9

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: src/20_Components/amp-selector.html [hunk 1]
```diff
@@ -14,138 +14,181 @@
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <!-- ## Setup -->
   <!--
-    Import the amp-selector component.
+    Import the `amp-selector` component.
   -->
   <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
   <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
```
Comment (kul3r4) [inline src/20_Components/amp-selector.html]: should we add why we are adding those additional libraries?

File: src/20_Components/amp-mustache.html [hunk 2]
```diff
@@ -16,50 +16,50 @@
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <!-- ## Setup -->
   <!--
-    Import the `amp-mustache` tag.  You'll need to do this along with any other tags your use case requires.  For the purposes of this sample, we'll use `amp-list`.
+    Import the `amp-mustache` tag.  You'll need to do this along with any other tags your use case requires. 
   -->
   <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <!--
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply src/20_Components/amp-mustache.html]: done

File: src/20_Components/amp-selector.html (hunk 1 above)
Comment (sebastianbenz) [reply src/20_Components/amp-selector.html]: done

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

**Pull request merged**